auth_matrix/admin.py

```diff
@@ -61,38 +66,24 @@ class Meta:
         fields = ("name",)
         export_order = fields
 
-    def dehydrate(self, group):
-        data = super().dehydrate(group)
+    def after_export(self, queryset, data, *args, **kwargs):
         permission_names = [permission.name for permission in Permission.objects.all()]
         for permission_name in permission_names:
-            data[permission_name] = permission_name in [
-                permission.name for permission in group.permissions.all()
-            ]
-        return data
-
-    def export(self, queryset, *args, **kwargs):
-        dataset = super().export(queryset, *args, **kwargs)
-        permission_names = [permission.name for permission in Permission.objects.all()]
-        for permission_name in permission_names:
-            dataset.append_col(
+            data.append_col(
                 [
                     group.permissions.filter(name=permission_name).exists()
                     for group in queryset
                 ],
                 header=permission_name,
             )
```

Comment:
@alextreme I see that the permission `name` field is an English non-unique string created by django when the permission is created. I think it'd be better to use the `code_name` and `content_type` field together to report the permission, or at least use `str(permission)` which includes the content type?

In its current form, you could have the same/similar model name in different Django apps (as we do with `Log` from yubin/django-timeline-logger/admin log/...) and the permissions for a given group could be reported incorrectly, because the content-type is not taken into account.

This could yield a permission as `myapp_MyModel:can_view_mymodel`, so, I suppose you could clean up this string to `myapp:can_view_mymodel`. On the other hand, when I define custom permission `foo` on two different models in the same app, they are different permission records and they would both show as `myapp:foo`, so maybe `myapp:mymodel:foo` would be best to make it non-ambiguous